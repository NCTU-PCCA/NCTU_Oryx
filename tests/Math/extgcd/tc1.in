24
39
11299333
2832933
237372
1
