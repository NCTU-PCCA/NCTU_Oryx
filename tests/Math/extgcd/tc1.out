41666667
717948723
785868428
337739723
198780818
1
